pkg/validation/application_installation.go

```diff
@@ -69,6 +69,13 @@ func ValidateApplicationInstallationSpec(ctx context.Context, client ctrlruntime
 		}
 	}
 	allErrs = append(allErrs, ValidateDeployOpts(spec.DeployOptions, specPath.Child("deployOptions"))...)
+
+	// Ensure that not both values and ValuesBlock fields are set simultaneously
+	if len(ai.Spec.Values.Raw) > 0 && ai.Spec.ValuesBlock != "" {
```

Comment:
just thinking if we would need to validate ValuesBlock for Cilium the same way as we do for the Values

Reply:
That is a good point. I was looking into the cilium cni code for this and from my perspective we should be good. Because the cni code is going to create an AppInstall at the api-server. And since the validation runs for every AppInstall applied at the api-server, we would get back an error that bubbles up to the customer.